mozilla-mobile/firefox-ios
# Remove #9591 [v97]: Remove non-compliant search engines

# Overview
This PR addresses [this JIRA ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-3491) and #9591. 

## Testing
- See `list.json`
- Pick a locale that has edits (removal of an engine).
- Change your device's region and language to match the locale you selected.
- Begin a search from the URLBar (don't execute the search).
- You'll see one less option to select from quick search below.
